src/pages/use-less-energy/index.js

```diff
@@ -215,8 +216,9 @@ const UseLessEnergyPage = () => (
             <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
               Lastly don’t forget to{" "}
               <Link
-                to="/choose-renewable-energy#actions"
+                to="/choose-renewable-energy/use-less-energy/#actions"
```

Comment:
I believe this is a typo, as it's broken locally. I think it should remain `/choose-renewable-energy#actions`